.babelrc

```diff
@@ -18,30 +17,26 @@
             "and_ff >= 49",
             "ios_saf >= 10"
           ]
-        }
+        },
+        // converts `import "@babel/polyfill"` into multiple, smaller imports that
+        // polyfills only the features that are used, and only for the targeted browsers
+        "useBuiltIns": "usage"
```

Comment:
Let's discuss the possible values:

* `false` is the default, meaning `import "@babel/polyfill"` imports the entire polyfill module, which is pretty big.
* `"entry"` converts the import into multiple, smaller imports, importing the minimum polyfills required to polyfill [the targeted browsers](https://github.com/bbc/simorgh/pull/1312/files#diff-e56633f72ecc521128b3db6586074d2cL7) (e.g. `"last 2 chrome versions"`). See https://babeljs.io/docs/en/babel-preset-env#usebuiltins-entry
* `"usage"` takes it one step further by doing code analysis and only importing polyfills for features that are actually being used. It's classed as "experimental" but errs on the side of "polyfill too much" rather than "too little". In any case, I don't see how it's possible that it would create a bigger bundle than `"entry"` or `false`, so is a good option to choose (and is [the way we're going in Visual Journalism](https://github.com/bbc/news-vj-developer-scaffold/pull/587)). See https://babeljs.io/blog/2018/08/27/7.0.0#automatic-polyfilling-experimental